multicore_tsne/tsne.cpp

```diff
@@ -491,18 +491,8 @@ void TSNE::zeroMean(double* X, int N, int D) {
 }
 
 
-// Generates a Gaussian random number
-double TSNE::randn() {
-    double x, y, radius;
-    do {
-        x = 2 * (rand() / ((double) RAND_MAX + 1)) - 1;
-        y = 2 * (rand() / ((double) RAND_MAX + 1)) - 1;
-        radius = (x * x) + (y * y);
-    } while ((radius >= 1.0) || (radius == 0.0));
-    radius = sqrt(-2 * log(radius) / radius);
-    x *= radius;
-    y *= radius;
-    return x;
+double TSNE::random() {
+    return rand() / ((double) RAND_MAX + 1);
```

Comment:
Why did you decide to switch from normal to uniform RV?

Reply:
Decide? You give me too much credit. 😄 I just tried to take every possible further optimization step I could find. And this function loops and calls sqrt and log; what appears to be, observing the results on the few datasets I ran it with, completely redundant.

But seeing now how simplification adds no real speed advantage, I'm happy to purge it.